### PR TITLE
add: command transfer ownership

### PR DESCRIPTION
**Pull request by arivictor**
Changes:

- Add command `transferOwnership` in `hardhat.config.ts`

### CODE REVIEW

**Review by channing-magiceden — APPROVED** [review]

**Review by nothing0012 — APPROVED** [review]

**Comment by arivictor** [comment]
Hey @nothing0012 looks like PR waiting on workflow approval, just following up.

https://github.com/magiceden-oss/erc721m/actions/runs/3373888146

**Pull request merged**
